src/main/java/com/plugish/woominecraft/WooMinecraft.java

```diff
@@ -250,4 +251,44 @@ public void wmc_log( String message, Integer level ) {
 	public void initCommands() {
 		getCommand( "woo" ).setExecutor( new WooCommand() );
 	}
+

```

Comment:
I'm all for set/get methods, but some of these seem overkill? What benefit do we get from creating method overhead just to access/set variables?

Historically methods are created as helper functions and do more than one line of code. It seems like most of the below methods were created for the sake of making a method... not for a real purpose.

To name a few:
* `getInstance()`
* `getLang()`
* `getL10n()`
* `getPluginConfig()`

I'm all for adding helper funcitons, but it's like these were added purely for the sake of 'remembering a function' rather than creating 'self-documenting variable names'.

I just cannot see the benefit of having this method overhead. It's just another set of instructions stacked on the memory. But again, I'm open to discussion here.